WcaOnRails/app/models/eligible_country_iso2_for_championship.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class EligibleCountryIso2ForChampionship < ApplicationRecord
```

Comment:
This name inferred form the table name that has been around in our discussion for a while, and that's what @jfly suggested in the issue description.
I agree that the name is not perfect, but for me `ChampionshipCountry` is not more clean either, cannot think of anything better than those though.